src/component.js

```diff
@@ -12,9 +12,7 @@ export let ENABLE_CLASSES = false;
  * @type {import('./internal').RendererState}
  */
 export const rendererState = {
```

Comment:
With something like https://github.com/preactjs/preact/pull/3493 in mind the state could still be a good option @developit